Lib/glyphsLib/builder/kerning.py

```diff
@@ -20,31 +20,47 @@
 
 def to_ufo_kerning(self):
     for master in self.font.masters:
+        master_id = master.id
         kerning_source = master.metricsSource  # Maybe be a linked master
         if kerning_source is None:
             kerning_source = master
-        if kerning_source.id in self.font.kerning:
-            kerning = self.font.kerning[kerning_source.id]
-            _to_ufo_kerning(self, self._sources[master.id].font, kerning)
+        if kerning_source.id in self.font.kerningLTR:
+            kerning = self.font.kerningLTR[kerning_source.id]
+            _to_ufo_kerning(self, self._sources[master_id].font, kerning)
+        if kerning_source.id in self.font.kerningRTL:
+            kerning = self.font.kerningRTL[kerning_source.id]
+            _to_ufo_kerning(self, self._sources[master_id].font, kerning, "RTL")
```

Comment:
since both LTR and RTL pairs end up in the single UFO kerning.plist dict, is a clash ever possible, and should be guard against that? If so, then they way this is coded means the last wins, so RTL duplicate sticks. I'm not 100% sure 1) it the clash is at all possible 2) what should we do about it

Reply:
A "clash" as in: A kerning pair that has different values in both directions?

In that case I need to [quote Simon](https://github.com/googlefonts/glyphsLib/pull/778#issuecomment-1335262059): _That will work for all cases except for situations where the same glyph pair has both LTR and RTL kerning, with different values in each table. But we were never going to get that right anyway._

Reply:
I mean a "clash" as in setting the same (first, second) key in the ufo.kerning dict as result of merging in both the LTR and RTL separate kerning dicts into the single dict. Maybe for the way we name the classes a situation where a class-class pair would clash is not possible, but a single-single pair might still clash.
At least a warning (right before overwriting a value) would help a user deal with it somehow

Reply:
Oh shite. Okay, I'll add a warning